### ISSUE TITLE
unable to load from url

### ISSUE THREAD

**Issue by sudipta213** [pycampers/flutter_pdf_viewer#3]
RaisedButton(
                 onPressed: () => FlutterPdfViewer.loadFilePath("https://firebasestorage.googleapis.com/v0/b/eduhitan.appspot.com/o/Online%20Recruitment%20Portal%20of%20APDCl.pdf?alt=media&token=54b81165-8570-448c-8e91-ff7db2410143"),
                 child: Text('OPEN'),
               ),

**Comment by hamidabdulmalik** [comment]
@devxpy , so where are we with this solution? When do we expect this to be ready?

**Comment by devxpy** [comment]
@hamidabdulmalik  I haven't looked into it yet.

Did you try my specified work-around? If yes, please share the code so I can test it out for myself or better yet, submit a pull request.

The OP didn't reply to my post, so it never occurred that this was an important feature to have in a pdf viewing lib.

The underlying native pdf viewing library doesn't support it, and there's good reason for it:

> Why I cannot open PDF from URL?
Downloading files is long running process which must be aware of Activity lifecycle, must support some configuration, data cleanup and caching, so creating such module will probably end up as new library.


Although, I think downloading a file in Dart should be straightforward. 

So TLDR; Please let me know if you have some experience downloading files in flutter, else wait for me to shell out some time.

Thanks!

**Comment by gedw99** [comment]
Yeah I also would have though downloading asynchronously to the file system
on the app and then opening the pdf from the same filesystem would be fine.
